FirebaseAuth/Tests/Sample/SwiftSample/GoogleAuthProvider.m

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright 2019 Google
```

Comment:
Could avoid duplicating these two files with ifdef's and a build setting in the Xcode project, but this could be fine, especially if they're going to evolve separately.

Reply:
Good to know, thanks! I'll leave it as is for now, as I expect there to be more changes.